.github/workflows/matrix.yml

```
@@ -9,83 +9,82 @@ jobs:
-        include:
-          - distro: debian
-            cross: null
-            runner: ubuntu-24.04-arm
-          - distro: debian
-            cross: i386
-            runner: ubuntu-latest
-          - distro: debian
-            cross: s390x
-            runner: ubuntu-24.04-arm
+        # NOTE: values are joined by '-' for name of container image, unless `CONTAINER` overrides it
+        - { DISTRO: arch, ARCH: amd64 }
```

actionlint (1.7.12)

[error] "matrix" section is sequence node but mapping node is expected

(syntax-check)